### ISSUE TITLE
I think an additional test to preclude the very bad solution below should be added

### ISSUE THREAD

**Issue by waltdakind** [FreeCodeCamp/FreeCodeCamp#10741]
Challenge [Check for Palindromes](https://www.freecodecamp.com/challenges/check-for-palindromes#?solution=%0Afunction%20palindrome(str)%20%7B%0A%20%20%0A%20%20%2F%2FHAHA%20cheating%20worked!%0Aif%20(str%20%3D%3D%20'not%20a%20palindrome'%20%7C%7C%20str%20%3D%3D%20'nope'%20%7C%7C%20str%20%3D%3D'almostomla'%20%7C%7C%20str%20%3D%3D%20'1%20eye%20for%20of%201%20eye.'%20%7C%7C%20str%20%3D%3D%20'five%7C%5C_%2F%7Cfour')%0A%7Breturn%20false%3B%7D%0A%20%20else%20%7B%0A%20%20%20%20return%20true%3B%0A%20%20%7D%0A%20%20%2F%2F%20Good%20luck!%0A%7D%0A%0A%0A%0Apalindrome(%22eye%22)%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/52.0.2743.116 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

function palindrome(str) {
  
  //HAHA cheating worked!
if (str == 'not a palindrome' || str == 'nope' || str =='almostomla' || str == '1 eye for of 1 eye.' || str == 'five|\_/|four')
{return false;}
  else {
    return true;
  }
  // Good luck!
}



palindrome("eye");

```

**Comment by sf-jd** [comment]
Well, what you should be learning from this is that your code will work for a reduced amount of cases, the moment you do something like this in a real life scenario, your client will be so up your ass with complaints about it not working for their users, you'll just decide to do it right the first time to cover as many use cases as possible ;)

**Comment by marhyorh** [comment]
Lol. You know that's not what you should be doing. I guess its time we add hidden test cases.

**Comment by dhcodes** [comment]
Closed as duplicate of #7600. Thanks.
